tests/tests.cabal

```diff
@@ -17,7 +17,7 @@ flag stack
                " buildable: False " by default
 
 flag measure-timings
-  default:     True
+  default:     False
```

Comment:
Revert.

Reply:
Actually, this makes sense to keep, no?  It makes the CI run faster it seems because it doesn't have to deal with ten billion lines of output.  Alternatively, I could filter it, but we don't _use_ the timings in the CI currently, so I'm tempted to just disable this flag unless you explicitly ask for it.

Reply:
It makes sense to me to disable it for now. But then you need to explain in the readme how to enable it to collect timings.